doc/ingest.md

```diff
@@ -26,6 +26,7 @@ type Advertisement struct {
     ContextID Bytes
     Metadata Bytes
     IsRm Bool
+    ExtendedProvider optional ExtendedMetadata
```

Comment:
Why nesting of `ExtendedMetadata` struct type? Why not `ExtendedProviders optional [ExtendedProvider]`? is there an explicit requirement for "family" grouping?